client/components/game-forms/CreateGameForm.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
```

Comment:
This should probably be called 'GameForm'...it will eventually be wrapped with a 'Create GameForm' which will create new games and an 'EditGameForm" which will handle edits, but also pass in initialvalues.